lldb/source/Plugins/ObjectFile/Minidump/ObjectFileMinidump.cpp

```diff
@@ -56,18 +56,18 @@ size_t ObjectFileMinidump::GetModuleSpecifications(
 }
 
 bool ObjectFileMinidump::SaveCore(const lldb::ProcessSP &process_sp,
-                                  const lldb_private::FileSpec &outfile,
-                                  lldb::SaveCoreStyle &core_style,
+                                  lldb_private::CoreDumpOptions &core_options,
                                   lldb_private::Status &error) {
   // Set default core style if it isn't set.
-  if (core_style == SaveCoreStyle::eSaveCoreUnspecified)
-    core_style = SaveCoreStyle::eSaveCoreStackOnly;
+  if (core_options.GetCoreDumpStyle() == SaveCoreStyle::eSaveCoreUnspecified)
+    core_options.SetCoreDumpStyle(SaveCoreStyle::eSaveCoreStackOnly);
```

Comment:
We should move this code into the `PluginManager::SaveCore` function and set this correctly prior to call each instance.save_core, that way each plug-in won't do things differently.